src/main/java/org/reaktivity/nukleus/kafka/internal/KafkaNukleusFactorySpi.java

```diff
@@ -46,12 +46,20 @@
 import org.reaktivity.nukleus.kafka.internal.types.OctetsFW;
 import org.reaktivity.nukleus.kafka.internal.types.control.KafkaRouteExFW;
 import org.reaktivity.nukleus.kafka.internal.types.control.RouteFW;
+import org.reaktivity.nukleus.kafka.internal.util.DelayedTaskScheduler;
 
 public final class KafkaNukleusFactorySpi implements NukleusFactorySpi, Nukleus
 {
     public static final String MESSAGE_CACHE_BUFFER_ACQUIRES = "message.cache.buffer.acquires";
     public static final String HISTORICAL_FETCHES = "historical.fetches";
     private static final String MESSAGE_CACHE_BUFFER_RELEASES = "message.cache.buffer.releases";
+    public static final String IDLE_REQUESTS = "idle.requests";
+    public static final String METADATA_TIMEOUTS_SCHEDULED = "metadata.timeouts.scheduled";
+    public static final String METADATA_TIMEOUTS_CANCELLED = "metadata.timeouts.cancelled";
+    public static final String CONFIGS_TIMEOUTS_SCHEDULED = "describe.configs.timeouts.scheduled";
+    public static final String CONFIGS_TIMEOUTS_CANCELLED = "describe.configs.timeouts.cancelled";
+    public static final String FETCH_TIMEOUTS_SCHEDULED = "fetch.timeouts.scheduled";
+    public static final String FETCH_TIMEOUTS_CANCELLED = "fetch.timeouts.cancelled";
```

Comment:
Normal operation will continuously schedule and cancel timers, but the unusual event is to timeout due to idle.

Perhaps the following counters would be more targeted?
```
metadata.request.idle.timeouts
describe.configs.request.idle.timeouts
list.offsets.request.idle.timeouts
fetch.request.idle.timeouts
```